apps/coscrad-frontend/src/store/index.ts

```diff
@@ -12,6 +14,7 @@ export const rootReducer = combineReducers({
     [TAGS]: tagReducer,
     [NOTES]: noteReducer,
     [CATEGORY_TREE]: categoryTreeReducer,
+    [TERMS]: termReducer,
 });
 
 export const setupStore = (preloadedState?: PreloadedState<RootState>) =>
```

Comment:
We could read some JSON that is exported from the backend by mapping test-data domain models through views layer and use this to pre-populate Redux for tests...